docs/core/compatibility/unsupported-apis.md

```diff
@@ -184,6 +184,7 @@ This article organizes the affected APIs by namespace.
 
 | Member | Platforms that throw |
 | - | - |
+| <xref:System.Runtime.InteropServices.IDispatchImplAttribute> | All |
```

Comment:
Not sure if I need to add this here as most of the other APIs throw the exception at run time.

Reply:
This will throw at runtime too, but during loading the assembly with this attribute. I think this makes sense here.